ee/pages/api/integrations/stripepayment/webhook.ts

```diff
@@ -5,6 +5,7 @@ import Stripe from "stripe";
 import stripe from "@ee/lib/stripe/server";
 
 import { CalendarEvent } from "@lib/calendarClient";
+import { IS_PRODUCTION } from "@lib/config/constants";
```

Comment:
Do we want this to mean "production build" or "production site"? Technically all preview deploy are production builds

Reply:
Preview will be production in this context.

The variable just does `process.env.NODE_ENV === 'production'`... I think it's a bit unnecessary to have this shortcut because it creates this sort confusion.

Reply:
We can leave it as it was before here if you think it creates confusion.

Reply:
You can take a look of the variable I've [made in Cal.com website](https://github.com/calendso/website/blob/main/lib/utils/isProductionSite.ts) hence the question. IDK if you we're trying to achieve the same thing. If it this is just for "production builds" then it doesn't makes much sense.

I would rather:
a. Rename the constant to `isProductionBuild` or
b. Keep it as `process.env.NODE_ENV === 'production'` since we're already familiar with it.

Legibility over all things IMO.